src/App.vue

```diff
@@ -107,7 +107,7 @@
                 <!-- Corrections still use the basic Fee Summary component -->
                 <aside v-if="isCorrectionFiling">
                   <SbcFeeSummary
-                    :filingData="[...getFilingData]"
+                    :filingData="getFilingData"
```

Comment:
So there is a significant difference between the old code and the new code.

Reply:
I think you should revert this.

The old code takes all properties in getFilingData, spreads them, and puts them in a new array. The new code just passes in a reference (pointer) to the getFilingData object (not an array).

Create UI passes this prop into SbcFeeSummary indirectly using a local getter (which itself uses the Store getter), which [returns an array](https://github.com/bcgov/business-create-ui/blob/da55ed1951a58cb88b18da309cf44f8a3133b8c4/src/App.vue#L339).

So this prop needs to be changed back to an array.

Reply:
Okay I will revert the changes I made to this

Reply:
resolved